utils/lib/rr_graph/channel2.py

```diff
@@ -53,38 +52,74 @@ def __init__(self, tracks):
         self.trees = []
         self.tracks = sorted(tracks, key=lambda x: x[1] - x[0])
 
-    def _place_track(self, track):
-        """Add track to existing interval tree, if there is room (ie doesn't overlap)
-        If track won't fit in any existing tree, allocate a new tree for the track"""
-        for idx, tree in enumerate(self.trees):
-            if not tree.overlaps(track[0], track[1] + 1):
-                tree.add(
-                    Interval(begin=track[0], end=track[1] + 1, data=track[2])
-                )
-                return
+    def _start_track(self, track):
+        self.trees.append(list())
+        self.trees[-1].append(track)
```

Comment:
Isn't this equivalent to `self.trees.append([track])`?

Reply:
Ya, fixed.